docs/Installing.asciidoc

```diff
@@ -31,10 +31,19 @@ repository. Installation on openSUSE is therefore pretty simple:
 
 [source,sh]
 --------------------------------------------------------------------------------
-zypper ar -f obs://devel:openQA/openSUSE_13.2 openQA
-# needed for distros that lack some packages from Tumbleweed
-zypper ar -f obs://devel:openQA:13.2/openSUSE_13.2 openQA-perl-modules
-zypper in openQA
+
+- 13.2
+  zypper ar -f obs://devel:openQA/openSUSE_13.2 openQA
+  zypper ar -f obs://devel:openQA:13.2/openSUSE_13.2 openQA-perl-modules
+
+- Leap 42.1
+  zypper ar -f obs://devel:openQA/openSUSE_Leap_42.1 openQA
+  zypper ar -f obs://devel:openQA:42/openSUSE_Leap_42.1 openQA-perl-modules
```

Comment:
devel:openQA should fullfilled requirements. so far the only one missed perl module after Leap 42.1 released is perl-Critic ie. it's the only one package in devel:openQA:42 now, but it's a special case - do not need it to run openQA, unless you want to run cpanfile or make check. BTW perl-Critic does not build in devel:openQA:42 now...